interfaces/builtin/fwupd.go

```diff
@@ -209,15 +394,13 @@ const fwupdConnectedSlotAppArmor = `
 const fwupdPermanentSlotDBus = `
 <policy user="root">
     <allow own="org.freedesktop.fwupd"/>
+</policy>
+<policy context="default">
```

Comment:
should this still have the `deny own` part from previously?

Reply:
Sounds right. I will test that.

Reply:
Tested and changed.